index.bs

```diff
@@ -4257,6 +4264,11 @@ JavaScript APIs.
         <var ignore>allowCredentialDescriptorList</var> and set |output| to [TRUE]. The value of |appId| then replaces the `rpId`
         parameter of [=authenticatorGetAssertion=].
 
+Note: In practice, several implementations do not implement steps four and onward of the
+algorithm for [=determining if a caller's FacetID is authorized for an AppID=].
+Instead, in step three, the comparison on the host is relaxed to accept hosts on the
```

Comment:
What does "hosts on the same site" mean?  Do you mean subdomains, such as "onmicrosoft.com" also matching "mytenant.onmicrosoft.com"?

Reply:
The words `same site` should be linked in the generated HTML to [the WHATWG specification](https://url.spec.whatwg.org/#host-same-site) which defines this.